actions/complete/index.js

```diff
@@ -15,9 +15,9 @@ const { asFirefallAction } = require('../FirefallAction.js');
 
 async function main(params) {
   const {
-    prompt, temperature, model, firefallClient,
+    prompt, temperature, asJson, model, firefallClient,
   } = params;
-  return firefallClient.completion(prompt ?? 'Who are you?', temperature ?? 0.0, model ?? 'gpt-4');
+  return firefallClient.completion(prompt ?? 'Who are you?', temperature ?? 0.0, asJson ?? true, model ?? 'gpt-4-turbo');
```

Comment:
If `asJson` is `null` or `undefined`, it makes sense for the value to be `false`.